static/src/javascripts/projects/commercial/modules/commercial-features.js

```diff
@@ -72,18 +81,21 @@ class CommercialFeatures {
         this.videoPreRolls = this.dfpAdvertising;
 
         this.highMerch =
-            this.dfpAdvertising &&
+            (this.dfpAdvertising || this.adFree) &&
             !isMinuteArticle &&
             !isHosted &&
             !isInteractive &&
             !config.page.isFront &&
             !newRecipeDesign;
 
-        this.thirdPartyTags = externalAdvertising && !isIdentityPage;
+        this.thirdPartyTags =
+            (externalAdvertising || this.adFree) && !isIdentityPage;
 
         this.outbrain =
-            this.dfpAdvertising &&
+            (this.dfpAdvertising || this.adFree) &&
```

Comment:
Does this mean; outbrain on ad-free, and no outbrain on ads? (Presuming that `this.outbrain` is used to control showing outbrain or not!)